locale/en_US/LC_MESSAGES/clr-installer.po

```diff
@@ -543,6 +543,33 @@ msgstr "Installation in progress."
 msgid "Installation successful."
 msgstr "Installation successful."
 
+msgid " (default)"
+msgstr " (default)"
+
+msgid "Add Extra Arguments:"
+msgstr "Add Extra Arguments:"
+
+msgid "Remove Arguments:"
+msgstr "Remove Arguments:"
+
+msgid "Kernel Configuration"
+msgstr "Kernel Configuration"
+
+msgid ", custom command line arguments"
```

Comment:
Please drop the comma from the string. Instead, let's have two sentences.

Reply:
Done